### ISSUE TITLE
Game crashes on start

### ISSUE THREAD

**Issue by RudiTheStrange** [anegostudios/VintageStory-Issues#412]
Vintage Story 1.12.5
Windows 10 Pro ver 10.018363 build 18363
No (other) Mods

Trying to Start the Game (after successfully playing the Day before). It crashes at the loadingscreen by "Loading Mod Meta Infos"
No mods added. No changes made. No hardwaredriver changed. No Windows Updates made since last play. Tried to avoid the crash by reinstall Game. Still same problem :(

```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.12.5 (Stable)05.03.2020 08:38:40: Critical error occurred
System.NullReferenceException: Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.
   bei Vintagestory.Client.NoObf.ClientPlatformWindows.LoadTexture(IBitmap bmp, Boolean linearMag, Int32 clampMode, Boolean generateMipmaps)
   bei _mKeEWUbBZsjLS6VlGWgOoYQFk2K._KNs6g2GMtsTu5gnL4MXdRgZVlLh()
   bei _lt8TgSVuBImgzFgAFlSeVzSFsSy._HdDsdRJDH8nMF88r278C4eqdINGA()
   bei _lt8TgSVuBImgzFgAFlSeVzSFsSy._gjiBClIE7Yspuz65gCqPGSAnUYeb()
   bei _lt8TgSVuBImgzFgAFlSeVzSFsSy._NbPATb9BXlxRc4ZhH2GgyFCUInp(Single )
   bei Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   bei System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   bei OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   bei OpenTK.GameWindow.DispatchRenderFrame()
   bei OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   bei _hCuHXesCXlp0LGSdrsPFpw4up5n._mOm8EuKIvHpaJoztxNWRpAoOpbM(_55ISULxdcpPfF8D2JP831aWhOBe , String[] )
   bei _iCfIbY24GyBXP5lAwqsSlD9COxC._mOm8EuKIvHpaJoztxNWRpAoOpbM(ThreadStart )
-------------------------------
```

**Comment by PAT899** [comment]
same bro literally posted it right before you

**Comment by Monahven-de** [comment]
I have the exact same problem. Same log. Played yesterday for the first time, for quite a few hours. Today it crashes on startup every time I try.

**Comment by om3gaman92** [comment]
Here's the fix

https://www.vintagestory.at/blog.html/news/crash-on-startup-fix-v1126-r240/

**Comment by PAT899** [comment]
thx a bunch

**Comment by RudiTheStrange** [comment]
Yessss! :D thx. Time to play after work

**Comment by copygirl** [comment]
As mentioned, this was fixed in `1.12.6`.
